### ISSUE TITLE
start_folder value in step 5

### ISSUE THREAD

**Issue by GuptaDiva** [nicknochnack/ActionDetectionforSignLanguage#28]
shouldn't the value of the start_folder initialized with 0 rather than 30 since we are adding the no. sequences to it therefore resulting in frames accessing values greator than 30.

**Comment by aaronse** [comment]
Yes.  ```start_folder``` wasn't in the original video https://youtu.be/doDUihpj6ro?si=e9g7SAPgLxKe3h0_&t=3740, guessing Nicholas added during subsequent capture/training session.
